agent/03_agent_deployment.sh

```diff
@@ -13,6 +13,13 @@ source $SCRIPTDIR/agent/common.sh
 
 early_deploy_validation
 
+function add_host_dns_entries() {
+    ip_list=$1
+
+    sudo virsh net-update ostestbm add dns-host  "<host ip='${ip_list[0]}'> <hostname>master-0-static</hostname> </host>"  --live --config
```

Comment:
Doesn't the network name depend on the cluster name? I think there's a variable in common.sh.

Reply:
Yes changed to use variable for network name